docs/tutorial.md

```diff
@@ -1,3 +1,117 @@
-# Tutorial
+# Tutorial: using the AWS plugins for Backstage
 
-Coming soon!
+This tutorial walks you through an example of using the AWS plugins for Backstage.
+
+Tutorial steps:
+
+<!-- toc -->
+
+1. [Set up your Backstage app](#set-up-your-backstage-app)
+1. [Clone this repository](#clone-this-repository)
+1. [Create prerequisite AWS resources](#create-prerequisite-aws-resources)
+1. [Run the Backstage app locally](#run-the-backstage-app-locally)
+1. [Explore the Plugins](#explore-the-plugins)
+1. [Cleanup](#cleanup)
+<!-- tocstop -->
+
+## Set up your Backstage app
+
+This tutorial assumes that you have a working Backstage application. To use this tutorial with a local Backstage app, follow the main [Backstage instructions](https://backstage.io/docs/getting-started/create-an-app) to create and run a Backstage app locally.
+
+## Clone this repository
+
+Before we start, clone the repository to get access to the necessary sample code and configurations.
+
+```
+git clone https://github.com/awslabs/backstage-plugins-for-aws.git
+cd backstage-plugins-for-aws
+```
+
+## Create prerequisite AWS resources
+
+To get started, we'll deploy a sample AWS CDK project included in the repository. The CDK project simplifies setup by automatically creating sample ECS services and other resources. These resources allow you to interact with real data in the Backstage plugins without needing to configure everything manually.
+
+### Instructions
+
+1. Navigate to the CDK Project Directory
+
+   Begin by navigating to the directory containing the CDK project:
+
+   ```
+   cd examples/cdk
+   ```
+
+2. Install CDK Project Dependencies
+
+   Inside the directory, install the required dependencies:
+
+   ```
+   yarn install
+   ```
+
+3. Deploy the CDK Stack
+
+   Deploy the sample AWS resources defined in the CDK stack:
+
+   ```
+   cdk deploy
+   ```
+
+   This will provision the AWS resources needed for your project, such as ECS clusters and services.
+
+   Ensure your AWS credentials are configured properly before running the command.
+
+   Deployment may take a few minutes. Once completed, your AWS resources will be ready for use.
+
+## Run the Backstage app locally
+
+With the sample resources in place, you can now run the Backstage app locally to see the plugins in action. The repository already includes pre-installed and configured plugins, so no additional setup is required.
+
+Use the following command to start the Backstage app:
+
+```
+yarn dev
+```
+
+Open your browser and then navigate to `http://localhost:3000/catalog/default/component/example-website`. You should see entities displayed as shown below: ![Example Website](./images/example-website.png)
```

Comment:
Lets lead them step by step to the screen instead. Get them to open `http://localhost:3000` then use screenshots to navigate to the entity.